tests/containers/bci_test.pm

```diff
@@ -102,7 +102,7 @@ sub run {
     assert_script_run('source bci/bin/activate') if ($bci_virtualenv);
 
     record_info('Run', "Starting the tests for the following environments:\n$test_envs");
-    assert_script_run("cd /root/BCI-tests && git fetch && git reset --hard");
+    assert_script_run("cd /root/BCI-tests && git pull --ff");
```

Comment:
I think that fetch and a hard reset is a "harder reset" than a pull, no?

Reply:
we want to use the "harder reset"?

Reply:
Yes we do. We only pre-cache it so we don't have to pull everything, but the tests should always have the current state from the BCI-Tests repository.

Therefore the hard reset so that it would even work when there are force pushes. With `pull` (especially `--ff`) the call would simply  fail.

Reply:
We're telling git basically: "Upstream always wins, just nuke everything what's local if it's out of sync".

Reply:
Ok, I agree.